### DISCUSSION TITLE
Kotlin Language Server integration

### DISCUSSION THREAD

**Discussion by yahorbarkouski** [TypeFox/monaco-languageclient#517] [Q&A] [answered]
Hello! I'm just curious if any of you have tried to integrate your beautiful language client with [kotlin-language-server](https://github.com/fwcd/kotlin-language-server)? Perhaps there are any examples or reference, that you have collected at this point, exists? Thank you so much

**Comment by kaisalmen** [comment, marked as answer]
HI @yahorbarkouski no we have not done this so far. A complex language server (even if it is beautiful) usually requires a bit more work on the client side as well. Look here for example:
https://github.com/fwcd/vscode-kotlin/blob/main/src/languageSetup.ts

Just integrating it is usually the simple part. Making it fully work requires a lot more work. That's why we haven't done this here, yet.
